client/my-sites/plugins/plugin-sections/style.scss

```diff
@@ -60,7 +60,22 @@
 	line-height: 21px;
 	word-wrap: break-word;
 	hyphens: auto;
-	margin: 32px 0;
+	margin-bottom: 32px;
+}
+
+.plugin-sections__header + .plugin-sections__content {
+	margin-top: 32px;
+}
+
+/* Remove top margin for the element if we don't render the header. */
+.plugin-sections__content:not(:has(+ .plugin-sections__header)) .plugin-sections__banner > img:first-child,
```

Comment:
nested styles look good for this case 🙈

Reply:
Ha, fair. I don't have a strong opinion on that. Maybe I'm old-school, but non-nested styles are easier to find and don't require any build magic.

Reply:
I think that, in this case, the complexity of the selector contributes most significantly to the eye strain, which is why I left the comment. I think I'll leave it as is, as [flattened CSS styles are preferred ](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md#:~:text=DON%27T%20nest%20selectors%20in%20general.%20Exceptions%20are%20%3Ahover/%3Afocus/%3A%3Abefore/.is%2Dmodifier%2C%20and%20alike.)based on our CSS coding standards.